msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerAuthServiceStrategy.java

```diff
@@ -309,4 +272,79 @@ static Bundle getRequestBundleForRemoveAccount(@NonNull final OperationParameter
+        return performAuthServiceOperation(parameters.getAppContext(),
+                new AuthServiceOperation<Boolean>() {
+                    @Override
+                    public Boolean perform(IMicrosoftAuthService service) throws BaseException, RemoteException {
+                        return MsalBrokerResultAdapter
+                                .deviceModeFromBundle(
+                                        service.getDeviceMode()
+                                );
+                    }
+
+                    @Override
+                    public String getOperationName() {
+                        return ":getDeviceModeWithAuthService";
+                    }
+                });
+    }
+
+    @WorkerThread
+    protected List<ICacheRecord> getCurrentAccountInSharedDevice(@NonNull final OperationParameters parameters) throws InterruptedException, ExecutionException, RemoteException, BaseException {
+        return performAuthServiceOperation(parameters.getAppContext(),
+                new AuthServiceOperation<List<ICacheRecord>>() {
+                    @Override
+                    public List<ICacheRecord> perform(IMicrosoftAuthService service) throws RemoteException, BaseException {
+                        return MsalBrokerResultAdapter
+                                .accountsFromBundle(
+                                        service.getCurrentAccount(
+                                                BrokerAuthServiceStrategy.getRequestBundleForGetAccounts(parameters)
+                                        ));
+                    }
+
+                    @Override
+                    public String getOperationName() {
+                        return ":getCurrentAccountInSharedDeviceWithAuthService";
+                    }
+                });
+    }
+
+    @WorkerThread
+    protected void signOutFromSharedDevice(@NonNull final OperationParameters parameters) throws BaseException, InterruptedException, ExecutionException, RemoteException {
+        final String methodName = ":signOutFromSharedDeviceWithAuthService";
+        performAuthServiceOperation(parameters.getAppContext(),
+                new AuthServiceOperation<Void>() {
+                    @Override
+                    public Void perform(IMicrosoftAuthService service) throws RemoteException, BaseException {
+                        final Bundle requestBundle = getRequestBundleForRemoveAccountFromSharedDevice(parameters);
+                        MsalBrokerResultAdapter.verifyRemoveAccountResultFromBundle(
+                                service.removeAccountFromSharedDevice(requestBundle)
+                        );
+
+                        return null;
+                    }
+
+                    @Override
+                    public String getOperationName() {
+                        return methodName;
```

Comment:
Again some places using `methodName` constant whereas others hard coding the operation name. I suggest picking one pattern.